src/sbtc-releases/sbtc-dev.md

```diff
@@ -1,3 +1,106 @@
 # sBTC Developer Release (0.1)
 
-Previously known as sBTC Mini. Details of what Mini is and how it differs to be added in [#13](https://github.com/stacks-network/sbtc-docs/issues/13).
+## Introduction
+
+The sBTC Developer Release implements a subset of the sBTC protocol. It aims
+to come as close as possible without requiring a hard fork. The Developer
+Release is implemented as a decentralized stacking pool on Stacks 2.4 that distributes rewards in sBTC. It means sBTC is secured by the locked STX in the stacking pool.
```

Comment:
Nit: Could we have a link here to the definition of a stacking pool or any resources where people can read and understand what a stacking pool is?

Reply:
@friedger what is the most comprehensive resource on Stacking pools these days? We have this page but it never mentions "pool" and talks about non-custodial delegation instead. https://wallet.hiro.so/wallet/faq#stacking

Reply:
Something like this? https://app.sigle.io/friedger.id.stx/UOvy85BCSD-bjlrv_6q74
and then this: https://blog.friedger.de/posts/O3JQEdDTuG5tktirZyBU2

Reply:
What is a Stacking Pool?

The Stacks blockchain uses the proof of transfer (PoX) consensus algorithm. It requires that Stack miners send Bitcoins to two randomly selected Bitcoin addresses each Bitcoin block. The more a miner sends, the higher the probability to win the Stacks block reward. The receiving Bitcoin addresses are picked from the list of Stackers. Stackers are users who hold a lot of Stacks and who are prepared to freeze their tokens for some time. The Bitcoin payouts are limited to 4000. These are called reward slots. Therefore, one stacker needs to provide 1/4000 of all frozen Stacks. If one stacker provides 2/4000 of all frozen Stacks, the user receives two payouts and so on. If users don't want to freeze the minimum amount, they can collaborate with other users and join a Stacking pool. One user, the pool admin, would receive the Bitcoins rewards for their collective stacking and distribute the rewards to the users. This happens according to the rules specific to each pool.

Note: "Joining a pool" is the same as "delegating Stacks."

Reply:
I think we better integrate this into `sbtc-docs` somewhere rather than linking to an external resource. Is your blog MIT licensed? 😉

Reply:
Everything else seems resolved and this is just a nit. Mind if I merge?